### ISSUE TITLE
Year Shows As 2016

### ISSUE THREAD

**Issue by NicksIdeaEngine** [freeCodeCamp/freeCodeCamp#12513]
Challenge [Friendly Date Ranges](https://www.freecodecamp.com/challenges/friendly-date-ranges#?solution=%2F*%20jshint%20esversion%3A%206%2C%20asi%3A%20true%2C%20laxcomma%3A%20true%20*%2F%0A%0Afunction%20makeFriendlyDates(arr)%20%7B%0A%20%20const%20date1%20%3D%20arr%5B0%5D.split('-')%0A%20%20%20%20%20%20%2C%20date2%20%3D%20arr%5B1%5D.split('-')%0A%20%20%0A%20%20%0A%20%20%0A%20%20return%20arr%3B%0A%7D%0A%0AmakeFriendlyDates(%5B'2016-07-01'%2C%20'2016-07-04'%5D)%3B%0A%0A%2F*%0A%20%20use%20ordinal%20dates%2C%20not%20cardinal%20(1st%2C%202nd%2C%20etc)%0A%20%20no%20redundant%20info.%20%20if%20date%20range%20ends%20less%20than%20a%20year%20from%20when%20it%20begins%2C%20do%20not%20display%20ending%20year%0A%20%20%0A*%2F) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36</code>.

No major problem, but the challenge tells us the current year is 2016.  I'm also not sure if the challenge will be checking in some way that considers the current year as 2016.

**Comment by atjonathan** [comment]
The wording for this part of the challenge description should be changed to prevent having to change it every year. That part could maybe just be removed, I'm not sure how I would reword this...
cc/ @freeCodeCamp/moderators

**Comment by ooade** [comment]
#12425

**Comment by ooade** [comment]
#12425

**Comment by atjonathan** [comment]
@ooade, thanks for pointing this out, marking as duplicate 👍
